backend/db/ddl-1-column_types.jinja.sql

```diff
@@ -0,0 +1,11 @@
+-- Column data types ---------------------------------------------------------------------------------------------------
+ALTER TABLE {{schema}}concept_set_container ALTER COLUMN project_id TYPE text;
```

Comment:
@hugwuoke (FYI @Sigfried ) Yeah, I'm super stumped here as to what's going on with the diff here.
<img width="378" alt="Screen Shot 2023-06-09 at 6 28 52 PM" src="https://github.com/jhu-bids/TermHub/assets/13045020/90d62f5f-9ee2-464a-b677-0ec028e97ca5">

Basically all these files and many more that I'm seeing in "Files Changed" were already added to `develop` the other day when I merged my DB Refresh PR. So they should not be considered new additions. As far as I can see, `test_backend.py` should be the only change here.

To make things even weirder, I just fetched the latest updates from both these branches, and I would expect my local diff to be the same as what's shown here, or maybe just show `test_backend.py` hopefully, but instead it shows something completely different!

`git diff develop --name-only`
```
.github/workflows/db_refresh.yml
.github/workflows/refresh_db.yml
backend/app.py
backend/db/analysis.py
backend/db/config.py
backend/db/ddl.jinja.sql
backend/db/full_data_refresh.py
backend/db/initialize.py
backend/db/load.py
backend/db/utils.py
db_backup.sh
docs/backend/db/analysis.md
docs/developer.md
enclave_wrangler/models.py
enclave_wrangler/objects_api.py
enclave_wrangler/utils.py
frontend/package-lock.json
frontend/package.json
frontend/src/App.js
frontend/src/components/State.js
frontend/src/pages/AboutPage.js
frontend/src/pages/CsetComparisonPage.js
frontend/src/supergroup
test/test_backend.py
test/test_enclave_wrangler.py
```

This is soooo weird. I'd love to figure what's going on, but I'm not sure how easy that will be. @hugwuoke I should be available around 2:30pm on Monday. I think I would like to teach you `git rebase`, a `git merge` alternative. It's is something even experienced programmers I know find tricky, but I think I can explain it in a way that you will understand, and I think you will be able to use it effectively. Basically I think we'll want to maybe create a new PR and copy over your changes there, and I can teach you how to use `rebase` to keep branches up-to-date.